HW4_Uzun/AAmigo.py

```diff
@@ -0,0 +1,181 @@
+def protein_mass(seq: str):
+    """
+
+    Calculate the mass (Da) of a protein based on its amino acids sequence.
+    Takes a string of amino acids, returns the molecular weight in Da.
+    Amino acids in the string should be indicated as one-letter symbols.
+
+    """
+    aa_seq = list(seq.upper())
+    mass_dictionary = dict({'A': 89, 'R': 174, 'N': 132, 'D': 133, 'C': 121, 'Q': 146, 'E': 147, 'Z': 147,

```

Comment:
1) Не нужно тут писать dict, у нас уже {... : ...} это словарь
2) Такие вещи лучше выносить в начало кода. Это называют константами и нейминг у них идет целиком капсом. Мы это подробно обсуждали на консультации 03.10.23, советую посмотреть

Reply:
И не очень принято добавлять тип переменной в ее название. Хотя бы не целиком, а `dict`. Ну а вообще питонисто было бы как-то так:
```suggestion
    AA_MASSES: dict = dict({'A': 89, 'R': 174, 'N': 132, 'D': 133, 'C': 121, 'Q': 146, 'E': 147, 'Z': 147,
```